docs/reference/configuration.md

```diff
@@ -232,6 +232,21 @@ secrets:
         -----END EC PRIVATE KEY-----
 ```
 
+### `secrets.encryption{_file}`
+
+The encryption secret used for encrypting cookies and database fields. It takes
+the form of a 32-bytes-long hex-encoded string. To provide the encryption secret
```

Comment:
Hex-encoded is right for the string variant, not for the file variant though? Could you clarify that? Also it would be nice to have instructions on how to generate such file, e.g. `openssl rand 32 -out file`

Reply:
I changed the code to match the documentation by expecting hex characters in the file. This better matches behavior of similar features and makes file generation trivial.